net-nntp/sabnzbd/sabnzbd-3.6.1.ebuild

```diff
@@ -0,0 +1,147 @@
+
+COMMON_DEPS="
+	acct-user/sabnzbd
+	acct-group/sabnzbd
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/cheetah3[${PYTHON_USEDEP}]
+		dev-python/cherrypy[${PYTHON_USEDEP}]
+		dev-python/configobj[${PYTHON_USEDEP}]
+		dev-python/cryptography[${PYTHON_USEDEP}]
+		>=dev-python/feedparser-6.0.10[${PYTHON_USEDEP}]
+		>=dev-python/guessit-3.1.0[${PYTHON_USEDEP}]
+		dev-python/notify2[${PYTHON_USEDEP}]
+		dev-python/portend[${PYTHON_USEDEP}]
+		dev-python/puremagic[${PYTHON_USEDEP}]
+		>=dev-python/sabyenc-5.4.2[${PYTHON_USEDEP}]
+	')
+"
+
+DEPEND="${COMMON_DEPS}"
+
+RDEPEND="
+	${COMMON_DEPS}
+	>=app-arch/par2cmdline-0.4
+	net-misc/wget
+	7za? ( app-arch/p7zip )
+	rar? ( || ( app-arch/unrar app-arch/rar ) )
+	unzip? ( >=app-arch/unzip-5.5.2 )
+"
+
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/flaky[${PYTHON_USEDEP}]
+			dev-python/pkginfo[${PYTHON_USEDEP}]
+			dev-python/pyfakefs[${PYTHON_USEDEP}]
+			dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+			dev-python/pytest-httpserver[${PYTHON_USEDEP}]
+			dev-python/selenium[${PYTHON_USEDEP}]
+			dev-python/xmltodict[${PYTHON_USEDEP}]
+		')
+		www-apps/chromedriver-bin
+	)
+"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	MY_HOMEDIR="/var/lib/${PN}"
+	python-single-r1_pkg_setup
+}
+
+src_install() {
+	local d
+
+	for d in email icons interfaces locale po sabnzbd scripts tools; do
+		insinto "/usr/share/${PN}/${d}"
+		doins -r ${d}/*
+	done
+
+	exeinto "/usr/share/${PN}"
+	doexe SABnzbd.py
+
+	python_fix_shebang "${ED}/usr/share/${PN}"
+	python_optimize "${ED}/usr/share/${PN}"
+
+	newinitd "${FILESDIR}/${PN}-r1.initd" "${PN}"
+	newconfd "${FILESDIR}/${PN}.confd" "${PN}"
+
+	diropts -o "${PN}" -g "${PN}"
+	dodir "/etc/${PN}"
+	keepdir "/var/log/${PN}"
+
+	insinto "/etc/${PN}"
+	insopts -m 0600 -o "${PN}" -g "${PN}"
+	newins "${FILESDIR}"/${PN}-r1.ini ${PN}.ini
+
+	dodoc ISSUES.txt README.mkd
+
+	systemd_newunit "${FILESDIR}"/sabnzbd_at.service 'sabnzbd@.service'
+}
+
+src_test() {
+	EPYTEST_IGNORE=(
+		# Requires dev-python/tavern which is not currently packaged for Gentoo
+		tests/test_functional_api.py
+	)
+	epytest
+}
+
+pkg_postinst() {
+	if [[ -z "${REPLACING_VERSIONS}" ]]; then
+		# This is a new installation
+		einfo "Default directory: ${MY_HOMEDIR}"
+		einfo
+		einfo "To add a user to the sabnzbd group so it can edit SABnzbd+ files, run:"
+		einfo
+		einfo "    usermod -a -G sabnzbd <user>"
+		einfo
+		einfo "By default, SABnzbd will listen on TCP port 8080."
+	else
+		local v
+		for v in ${REPLACING_VERSIONS}; do
+			if ver_test "${v}" -lt 3; then
+				ewarn
+				ewarn "Due to changes in this release, the queue will be converted when ${PN}"
+				ewarn "is started for the first time. Job order, settings and data will be"
+				ewarn "preserved, but all jobs will be unpaused and URLs that did not finish"
+				ewarn "fetching before the upgrade will be lost!"
+				ewarn
+				break
+			fi
+		done
+	fi
```

Comment:
When was 3.0 added? I wonder if it's time to get rid of this message already.

Reply:
long time ago, removing.